server/gateway/src/childLoader.ts

```diff
@@ -123,8 +123,7 @@ class KeyValueLoader {
 
     private async attach(loader: Loader, docUrl: string) {
         const response = await loader.request({ url: docUrl });
-        if (response.status !== 200 ||
```

Comment:
Something to consider: There is requestFluidObject() that does almost same thing, and it hides / abstracts away status code and mimeType (see also https://github.com/microsoft/FluidFramework/issues/4194).
It would be great for 99% of the code in our repo not to care how responses are formed and use higher-level constructs. The only difference between this code and what requestFluidObject does is that latter throws while this code returns undefined.
We could create requestFluidObjectSilent (some other better name goes here :)) that does what requestFluidObject  but instead of exception returns undefined.

@markfields somewhere (can't find an issue) proposed similar thing - leveraging built into language typeof operator to differentiate object vs. text (as an example).